### DISCUSSION TITLE
Writing ECU Problem, Renault Scenic 3

### DISCUSSION THREAD

**Discussion by k3ve** [cedricp/ddt4all#1134] [Q&A]
Hello everyone!

I am trying to enable Android Auto/Apple CarPlay on my Renault Scenic 3. I work at a Renault dealer, so if I can make this work, it could be a great tool in our daily business.

What I use:
- R-Link version 11.346(newest)
- Laptop with Windows x64
- DTT4ALL-Windows-Installer-v3.0.1_Stable-x64
- DDT2000 Database Update 10.02.2021
- OBDLink SX
- Adapter cable with pin 3 to pin 13 and pin 11 to pin 12

I connect the laptop to the car, start DDT4ALL, select OBDLink SX, and hit Editor Mode. In the program, I select my car (Megane 3), then go to Navigation, MFD 5.4, and configuration 2. I scroll down to the right parameter, hit Expert Mode, change from 'not present' to 'present', and hit 'write ECU'. Some lines appear at the bottom, but nothing happens.

What am I doing wrong?

Cheers,
k3ve

![doesnt write](https://github.com/cedricp/ddt4all/assets/117689928/ee471ba1-68e1-440c-8269-610d0adb9320)

[Neues Textdokument.txt](https://github.com/user-attachments/files/15586897/Neues.Textdokument.txt)

**Comment by cedricp** [comment]
It looks like you're in "edition mode" , try "connected mode" instead at startup

**Reply by k3ve** [reply]
That did the trick!

Thanks @cedricp
